l10n/de/security/manager/chrome/pipnss/pipnss.properties

```diff
@@ -292,34 +292,6 @@ CertInfoPurposes=Zwecke
 CertInfoEmail=E-Mail
 CertInfoStoredIn=Gespeichert in:
 P12DefaultNickname=Importiertes Zertifikat
-<<<<<<< HEAD
-VerifyExpired=<Abgelaufen>
-VerifyRevoked=<Widerrufen>
-VerifyNotTrusted=<Nicht vertrauenswürdig>
-VerifyIssuerNotTrusted=<Aussteller nicht vertrauenswürdig>
-VerifyIssuerUnknown=<Aussteller unbekannt>
-VerifyInvalidCA=<Ungültige CA>
-VerifyDisabledAlgorithm=<Signatur-Algorithmus nicht sicher>
-VerifyUnknown=<Unbekannt>
-CertUser=Dein Zertifikat
-CertCA=CA (Zertifizierungsstelle)
-CertSSL=SSL-Server
-CertEmail=Peer S/MIME
-||||||| merged common ancestors
-VerifyExpired=<Abgelaufen>
-VerifyRevoked=<Widerrufen>
-VerifyNotTrusted=<Nicht vertrauenswürdig>
-VerifyIssuerNotTrusted=<Aussteller nicht vertrauenswürdig>
-VerifyIssuerUnknown=<Aussteller unbekannt>
-VerifyInvalidCA=<Ungültige CA>
-VerifyDisabledAlgorithm=<Signatur-Algorithmus nicht sicher>
-VerifyUnknown=<Unbekannt>
-CertUser=Ihr Zertifikat
-CertCA=CA (Zertifizierungsstelle)
-CertSSL=SSL-Server
-CertEmail=Peer S/MIME
-=======
```

Comment:
Did you check if they just moved these strings to another file?
If `CertUser` still exists somewhere, we'd better keep our edit.